roles/reproducer/tasks/ci_job.yml

```diff
@@ -84,7 +84,7 @@
           vars:
             zuul_params_filtered: >-
               {{
-                zuul | dict2items |
+                zuul_vars | dict2items |
```

Comment:
I think it should be `zuul_vars.zuul` actually

weird I didn't see it :/

Reply:
Yeah I think your right, testing